mappings/net/minecraft/structure/Structure.mapping

```diff
@@ -115,6 +121,40 @@ CLASS net/minecraft/class_3499 net/minecraft/structure/Structure
 		ARG 2 rotation
 		ARG 3 pivot
 		ARG 4 mirror
+	METHOD method_28053 (Lnet/minecraft/class_3499$class_3501;)I
+		ARG 0 blockInfo
+	METHOD method_28054 categorize (Lnet/minecraft/class_3499$class_3501;Ljava/util/List;Ljava/util/List;Ljava/util/List;)V
+		COMMENT Categorizes {@code blockInfo} based on its properties, modifying
+		COMMENT the passed lists in-place.
+		COMMENT
+		COMMENT <p>If the block has an NBT associated with it, then it will be
+		COMMENT put in {@code blocksWithNbt}. If the block does not have an NBT
+		COMMENT associated with it, but is always a full cube, then it will be
+		COMMENT put in {@code fullBlocks}. Otherwise, it will be put in
+		COMMENT {@code otherBlocks}.
+		COMMENT
+		COMMENT @apiNote After all blocks are categorized, {@link combineSorted}
+		COMMENT shoud be called with the same parameters to get the final list.
+		ARG 0 blockInfo
+		ARG 1 fullBlocks
+		ARG 2 blocksWithNbt
+		ARG 3 otherBlocks
+	METHOD method_28055 combineSorted (Ljava/util/List;Ljava/util/List;Ljava/util/List;)Ljava/util/List;
+		COMMENT {@return the list that sorts and combines the passed block lists}
+		COMMENT
+		COMMENT @apiNote The parameters passed shoule be the same one that was passed
+		COMMENT to previous calls to {@link categorize}. The returned value is meant to
+		COMMENT be passed to {@link PalettedBlockInfoList}.
```

Comment:
```suggestion
		COMMENT be passed to {@link #PalettedBlockInfoList}.
```

Reply:
hmm, don't think you use hash for nested or inner classes